OpenBLAS 0.3.29 make install failed

OS is Ubuntu 24.04. After running make and passing all the tests make install failed with this error

```
/usr/bin/ld: warning: /tmp/ccQ0lePk.o: missing .note.GNU-stack section implies executable stack
/usr/bin/ld: NOTE: This behaviour is deprecated and will be removed in a future version of the linker
make -f Makefile.install install
make[1]: Entering directory '/home/mb/src/OpenBLAS-0.3.29'
Makefile.install:318: *** missing separator.  Stop.
make[1]: Leaving directory '/home/mb/src/OpenBLAS-0.3.29'
make: *** [Makefile:427: install] Error 2
```

OpenBLAS 0.3.28 did install with no problem. I checked Makefile.install in both versions and they appear to be very different.

--- a/Makefile.install
+++ b/Makefile.install
@@ -315,8 +315,8 @@ endif
 
 endif
 ifeq ($(CPP_THREAD_SAFETY_TEST), 1)
-@install -m 666 cpp_thread_test/dgemm_tester $(DESTDIR)$(OPENBLAS_BINARY_DIR)
-@install -m 666 cpp_thread_test/dgemv_tester $(DESTDIR)$(OPENBLAS_BINARY_DIR)
+	@install -m 666 cpp_thread_test/dgemm_tester $(DESTDIR)$(OPENBLAS_BINARY_DIR)
+	@install -m 666 cpp_thread_test/dgemv_tester $(DESTDIR)$(OPENBLAS_BINARY_DIR)
 endif
 endif
 

